test/e2e/e2e_suite_test.go

```diff
@@ -63,6 +63,9 @@ var (
 	// If it is not set, a local clusterctl repository (including a clusterctl config) will be created automatically.
 	clusterctlConfig string
```

Comment:
Please change l.198 to use "KUBERNETES_VERSION" instead of "INIT_WITH_KUBERNETES_VERSION"

Reply:
Yes, it is straightforward to use the same version for bootstrap and workload cluster. I will remove it.